packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -148,26 +149,42 @@ const extractAttachmentUrl = (attachment: ChatAttachment): string => {
   return attachment.url || '';
 };
 const processChatMessageContent = (message: ChatMessageWithStatus): string | undefined => {
+  let content = message.content?.message;
   /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
-  if (sanitizedMessageContentType(message.type).includes('html') && message.content?.attachments) {
+  if (content && sanitizedMessageContentType(message.type).includes('html') && message.content?.attachments) {
```

Comment:
Would it be better for performance to check the presence of attachments first?

Reply:
Reordered